src/twisted/persisted/test/test_styles.py

```diff
@@ -91,6 +92,10 @@ def test_instanceBuildingNamePresent(self):
         self.assertEqual(m, foo.method)
         self.assertIsNot(m, foo.method)
 
+    if _PYPY:
+        test_instanceBuildingNamePresent.skip = (
+            "Does not work on PYPY")
```

Comment:
See [Twisted ticket 9239](https://twistedmatrix.com/trac/ticket/9239) and [PyPy ticket 2619](https://bitbucket.org/pypy/pypy/issues/2619/) should be fixed upstream released in PyPy 5.8.1